### PR TITLE
Add Ruby Together CTA

### PR DESCRIPTION
**Pull request by mrb**
/c @indirect

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,18 @@ See [bundler.io](http://bundler.io) for the full documentation.
...
 For help with common problems, see [ISSUES](https://github.com/bundler/bundler/blob/master/ISSUES.md).
 
+### Supporting
+
+<div id="cto_container" style="width:100%;">
+  <div id="rt_logo" style="width:20%; float:left;">
+    <a href="https://rubytogether.org/"><img src="assets/ruby_together_logotype_200.png"></a>
+  </div>
+  <div id="rt_description" style=" width:80%; float:left;">
+    Bundler is maintained by <a href="https://rubytogether.org/">Ruby Together</a>, a grassroots initiative committed to supporting the critical Ruby infrastructure you rely on. Contribute today <a href="https://rubytogether.org/developers">as an individual</a> or even better, <a href="https://rubytogether.org/companies">as a company</a>, and ensure that Bundler, Rubygems, and other shared tooling is around for years to come.
```
Comment (segiddins) [inline README.md]: `RubyGems`

**Comment by mrb** [comment]
@segiddins Fixed!

**Comment by parndt** [comment]
The logo (with its white background) looks a bit strange - is there a transparent background version that can be used here?

**Comment by indirect** [comment]
It would probably even be fine to hotlink the ruby together logo from the website. either way, I think transparent background would be better 👍

**Comment by indirect** [comment]
(It also looks like we need to exclude any graphics assets from the quality spec, which expects only text)

**Comment by mrb** [comment]
@parndt @indirect Ok! Hotlinked the SVG and zapped the `assets/` dir and file.

**Comment by indirect** [comment]
Thanks!

**Comment by indirect** [comment]
@homu r+

**Pull request merged**